Documentation/manual/kcc-design/customization-user-story.md

```diff
@@ -0,0 +1,138 @@
+**Expanding on ‘Customization User’ Scenarios**
+
+**Example #1: Planar 3D Platformer Character Controller**
+
+————————————————————————————————————————————————————————————————————————
+
+User Story:
+
+As a Customization User with an existing project, I’m interested in having my development studio switch to OpenKCC from a paid asset for the following reasons
+- To take advantage of arbitrary colliders
+- To benefit from wide test coverage and reliable low-level code
+- To contribute to, learn from, and benefit from an Open Source code base
+
+While we are interested in the the higher level features of OpenKCC, as a pre-existing studio we already have our own high level code that enables us to reach a high velocity of output with our designers who are already accustomed to our tools
+
+Ideally, our game architecture programmer will be able to use the API of the KCCMovementEngine to make it compatible with our high level code which already handles things such as state switching, etc.
```

Comment:
Agreed, that's why it's important we disconnect the KCCMovementEngine from my existing state machine API!